torch/_inductor/fx_passes/quantization.py

```diff
@@ -519,6 +519,10 @@ def __init__(self, op_name: str, scalars_attr=None, algorithm_attr=None):
                 generate_pattern_with_unary(qlinear_pt2e_pattern, aten.relu.default),
                 dtype=original_pattern_output_dtype,
             ),
+            UnaryAttr("gelu", [], ""): generate_pattern_with_output_quant(
```

Comment:
We need to cover both `None` and `tanh` as `algorithm_attr`, right? Should we need to register it twice with different `algorithm_attr`?